pandas/tests/frame/test_constructors.py

```diff
@@ -2569,6 +2570,13 @@ def test_construction_from_set_raises(self, typ):
         with pytest.raises(TypeError, match=msg):
             Series(values)
 
+    def test_construction_from_ndarray_datetimelike(self):
```

Comment:
looks like this passes in master.  is there a motivating case?

Reply:
Yes, but it only passes because right now it goes through the BlockManager->ArrayManager conversion code (which will go away eventually). An initial version of this PR did this wrong, and there was no constructor test to catch it (I found it out because some arithmetic test then failed because of the wronly constructed dataframe, but having an explicit test for it is useful I think)

Reply:
makes sense, thanks